components/notion/actions/duplicate-page/duplicate-page.mjs

```diff
@@ -52,7 +53,8 @@ export default {
     };
 
     const results = await this.notion.createPage(page);
-    $.export("$summary", `Successfully created page with id ${results.id}`);
+    const pageTitle = results?.properties?.title?.title[0]?.plain_text || results.id;
+    $.export("$summary", `Successfully created the new page, "${pageTitle}"`);
```

Comment:
@michelle0927 sorry, small nitpick -- can you make the `pageTitle` a link? Looks like the page URL is returned in the response from Notion.